pkg/describe/describe.go

```diff
@@ -14,26 +14,26 @@ import (
 func DescribePackage(
 	ctx context.Context,
 	pkgName string,
-) (*v1alpha1.Package, *client.PackageStatus, *v1alpha1.PackageManifest, error) {
+) (*v1alpha1.Package, *client.PackageStatus, *v1alpha1.PackageManifest, *[]v1alpha1.PackageEntrypoint, error) {
```

Comment:
You don't need to return the array additionally, it is already contained in the returned `PackageManifest`. So there should be no changes necessary to this function.